fastly/dictionary_info.go

```diff
@@ -0,0 +1,56 @@
+package fastly
+
+import (
+	"fmt"
+	"time"
+)
+
+// DictionaryInfo represents a dictionary metadata response from the Fastly API.
+type DictionaryInfo struct {
+	// LastUpdated is the Time-stamp (GMT) when the dictionary was last updated.
+	LastUpdated time.Time `mapstructure:"last_updated"`
```

Comment:
Please update `time.Time` to a pointer type.

<details open>
<summary>In all other files we use pointer types:</summary>
<pre>
fastly/acl.go:  CreatedAt *time.Time `mapstructure:"created_at"`
fastly/acl.go:  UpdatedAt *time.Time `mapstructure:"updated_at"`
fastly/acl.go:  DeletedAt *time.Time `mapstructure:"deleted_at"`
fastly/acl_entry.go:    CreatedAt *time.Time `mapstructure:"created_at"`
fastly/acl_entry.go:    UpdatedAt *time.Time `mapstructure:"updated_at"`
fastly/acl_entry.go:    DeletedAt *time.Time `mapstructure:"deleted_at"`
fastly/backend.go:      CreatedAt           *time.Time `mapstructure:"created_at"`
fastly/backend.go:      UpdatedAt           *time.Time `mapstructure:"updated_at"`
fastly/backend.go:      DeletedAt           *time.Time `mapstructure:"deleted_at"`
fastly/bigquery.go:     CreatedAt         *time.Time `mapstructure:"created_at"`
fastly/bigquery.go:     UpdatedAt         *time.Time `mapstructure:"updated_at"`
fastly/bigquery.go:     DeletedAt         *time.Time `mapstructure:"deleted_at"`
fastly/billing.go:      StartTime *time.Time     `mapstructure:"start_time"`
fastly/billing.go:      EndTime   *time.Time     `mapstructure:"end_time"`
fastly/billing.go:      SentAt    *time.Time `mapstructure:"sent_at"`
fastly/blobstorage.go:  CreatedAt         *time.Time `mapstructure:"created_at"`
fastly/blobstorage.go:  UpdatedAt         *time.Time `mapstructure:"updated_at"`
fastly/blobstorage.go:  DeletedAt         *time.Time `mapstructure:"deleted_at"`
fastly/cache_setting.go:        CreatedAt      *time.Time         `mapstructure:"created_at"`
fastly/cache_setting.go:        UpdatedAt      *time.Time         `mapstructure:"updated_at"`
fastly/cache_setting.go:        DeletedAt      *time.Time         `mapstructure:"deleted_at"`
fastly/condition.go:    CreatedAt *time.Time `mapstructure:"created_at"`
fastly/condition.go:    UpdatedAt *time.Time `mapstructure:"updated_at"`
fastly/condition.go:    DeletedAt *time.Time `mapstructure:"deleted_at"`
fastly/decode_hooks.go:// stringToTimeHookFunc returns a function that converts strings to a time.Time
fastly/dictionary.go:   CreatedAt *time.Time `mapstructure:"created_at"`
fastly/dictionary.go:   UpdatedAt *time.Time `mapstructure:"updated_at"`
fastly/dictionary.go:   DeletedAt *time.Time `mapstructure:"deleted_at"`
fastly/dictionary_item.go:      CreatedAt *time.Time `mapstructure:"created_at"`
fastly/dictionary_item.go:      UpdatedAt *time.Time `mapstructure:"updated_at"`
fastly/dictionary_item.go:      DeletedAt *time.Time `mapstructure:"deleted_at"`
fastly/director.go:     CreatedAt *time.Time   `mapstructure:"created_at"`
fastly/director.go:     UpdatedAt *time.Time   `mapstructure:"updated_at"`
fastly/director.go:     DeletedAt *time.Time   `mapstructure:"deleted_at"`
fastly/director_backend.go:     CreatedAt *time.Time `mapstructure:"created_at"`
fastly/director_backend.go:     UpdatedAt *time.Time `mapstructure:"updated_at"`
fastly/director_backend.go:     DeletedAt *time.Time `mapstructure:"deleted_at"`
fastly/domain.go:       CreatedAt *time.Time `mapstructure:"created_at"`
fastly/domain.go:       UpdatedAt *time.Time `mapstructure:"updated_at"`
fastly/domain.go:       DeletedAt *time.Time `mapstructure:"deleted_at"`
fastly/event_logs.go:   CreatedAt   *time.Time             `jsonapi:"attr,created_at,iso8601"`
fastly/ftp.go:  CreatedAt         *time.Time `mapstructure:"created_at"`
fastly/ftp.go:  UpdatedAt         *time.Time `mapstructure:"updated_at"`
fastly/ftp.go:  DeletedAt         *time.Time `mapstructure:"deleted_at"`
fastly/gcs.go:  CreatedAt         *time.Time `mapstructure:"created_at"`
fastly/gcs.go:  UpdatedAt         *time.Time `mapstructure:"updated_at"`
fastly/gcs.go:  DeletedAt         *time.Time `mapstructure:"deleted_at"`
fastly/gzip.go: CreatedAt      *time.Time `mapstructure:"created_at"`
fastly/gzip.go: UpdatedAt      *time.Time `mapstructure:"updated_at"`
fastly/gzip.go: DeletedAt      *time.Time `mapstructure:"deleted_at"`
fastly/header.go:       CreatedAt         *time.Time   `mapstructure:"created_at"`
fastly/header.go:       UpdatedAt         *time.Time   `mapstructure:"updated_at"`
fastly/header.go:       DeletedAt         *time.Time   `mapstructure:"deleted_at"`
fastly/health_check.go: CreatedAt        *time.Time `mapstructure:"created_at"`
fastly/health_check.go: UpdatedAt        *time.Time `mapstructure:"updated_at"`
fastly/health_check.go: DeletedAt        *time.Time `mapstructure:"deleted_at"`
fastly/logentries.go:   CreatedAt         *time.Time `mapstructure:"created_at"`
fastly/logentries.go:   UpdatedAt         *time.Time `mapstructure:"updated_at"`
fastly/logentries.go:   DeletedAt         *time.Time `mapstructure:"deleted_at"`
fastly/papertrail.go:   CreatedAt         *time.Time `mapstructure:"created_at"`
fastly/papertrail.go:   UpdatedAt         *time.Time `mapstructure:"updated_at"`
fastly/papertrail.go:   DeletedAt         *time.Time `mapstructure:"deleted_at"`
fastly/papertrail.go:   CreatedAt         *time.Time `form:"created_at,omitempty"`
fastly/papertrail.go:   UpdatedAt         *time.Time `form:"updated_at,omitempty"`
fastly/papertrail.go:   DeletedAt         *time.Time `form:"deleted_at,omitempty"`
fastly/papertrail.go:   CreatedAt         *time.Time `form:"created_at,omitempty"`
fastly/papertrail.go:   UpdatedAt         *time.Time `form:"updated_at,omitempty"`
fastly/papertrail.go:   DeletedAt         *time.Time `form:"deleted_at,omitempty"`
fastly/request_setting.go:      CreatedAt        *time.Time           `mapstructure:"created_at"`
fastly/request_setting.go:      UpdatedAt        *time.Time           `mapstructure:"updated_at"`
fastly/request_setting.go:      DeletedAt        *time.Time           `mapstructure:"deleted_at"`
fastly/response_object.go:      CreatedAt        *time.Time `mapstructure:"created_at"`
fastly/response_object.go:      UpdatedAt        *time.Time `mapstructure:"updated_at"`
fastly/response_object.go:      DeletedAt        *time.Time `mapstructure:"deleted_at"`
fastly/s3.go:   CreatedAt         *time.Time   `mapstructure:"created_at"`
fastly/s3.go:   UpdatedAt         *time.Time   `mapstructure:"updated_at"`
fastly/s3.go:   DeletedAt         *time.Time   `mapstructure:"deleted_at"`
fastly/service.go:      CreatedAt     *time.Time `mapstructure:"created_at"`
fastly/service.go:      UpdatedAt     *time.Time `mapstructure:"updated_at"`
fastly/service.go:      DeletedAt     *time.Time `mapstructure:"deleted_at"`
fastly/service.go:      CreatedAt     *time.Time `mapstructure:"created_at"`
fastly/service.go:      UpdatedAt     *time.Time `mapstructure:"updated_at"`
fastly/service.go:      DeletedAt     *time.Time `mapstructure:"deleted_at"`
fastly/service.go:      DeletedAt *time.Time `mapstructure:"deleted_at"`
fastly/service.go:      CreatedAt *time.Time `mapstructure:"created_at"`
fastly/service.go:      UpdatedAt *time.Time `mapstructure:"updated_at"`
fastly/splunk.go:       CreatedAt         *time.Time `mapstructure:"created_at"`
fastly/splunk.go:       UpdatedAt         *time.Time `mapstructure:"updated_at"`
fastly/splunk.go:       DeletedAt         *time.Time `mapstructure:"deleted_at"`
fastly/sumologic.go:    CreatedAt         *time.Time `mapstructure:"created_at"`
fastly/sumologic.go:    UpdatedAt         *time.Time `mapstructure:"updated_at"`
fastly/sumologic.go:    DeletedAt         *time.Time `mapstructure:"deleted_at"`
fastly/syslog.go:       CreatedAt         *time.Time `mapstructure:"created_at"`
fastly/syslog.go:       UpdatedAt         *time.Time `mapstructure:"updated_at"`
fastly/syslog.go:       DeletedAt         *time.Time `mapstructure:"deleted_at"`
fastly/token.go:        CreatedAt   *time.Time `mapstructure:"created_at"`
fastly/token.go:        LastUsedAt  *time.Time `mapstructure:"last_used_at"`
fastly/token.go:        ExpiresAt   *time.Time `mapstructure:"expires_at"`
fastly/token.go:        ExpiresAt *time.Time `form:"expires_at,omitempty"`
fastly/vcl.go:  CreatedAt *time.Time `mapstructure:"created_at"`
fastly/vcl.go:  UpdatedAt *time.Time `mapstructure:"updated_at"`
fastly/vcl.go:  DeletedAt *time.Time `mapstructure:"deleted_at"`
fastly/vcl_snippets.go: CreatedAt *time.Time  `mapstructure:"created_at"`
fastly/vcl_snippets.go: UpdatedAt *time.Time  `mapstructure:"updated_at"`
fastly/vcl_snippets.go: DeletedAt *time.Time  `mapstructure:"deleted_at"`
fastly/vcl_snippets.go: CreatedAt *time.Time `mapstructure:"created_at"`
fastly/vcl_snippets.go: UpdatedAt *time.Time `mapstructure:"updated_at"`
fastly/version.go:      CreatedAt *time.Time `mapstructure:"created_at"`
fastly/version.go:      UpdatedAt *time.Time `mapstructure:"updated_at"`
fastly/version.go:      DeletedAt *time.Time `mapstructure:"deleted_at"`
fastly/waf.go:  LastPush          *time.Time `jsonapi:"attr,last_push,iso8601"`
fastly/waf.go:  CreatedAt                        *time.Time `jsonapi:"attr,created_at,iso8601"`
fastly/waf.go:  UpdatedAt                        *time.Time `jsonapi:"attr,updated_at,iso8601"`
fastly/waf.go:  CreatedAt                        *time.Time `jsonapi:"attr,created_at,omitempty,iso8601"`
fastly/waf.go:  UpdatedAt                        *time.Time `jsonapi:"attr,updated_at,omitempty,iso8601"`
fastly/waf.go:  LastPush *time.Time `jsonapi:"attr,last_push,omitempty,iso8601"`
</pre>
</details>